kubernetes/src/test/resources/kubernetes/kubernetes.tests.bom

```diff
@@ -89,3 +89,13 @@ brooklyn.catalog:
             kubectl get deployments | grep workload- | wc -l
           assertOut:
             contains: '0'
+        - type: org.apache.brooklyn.test.framework.TestHttpCall
+          name: "11. Check Prometheus UI is Reachable"
```

Comment:
One other minor comments: we should use `brooklyn.config:` rather than putting config at the top level. But I think you can get away with the top-level for entities (just not for enrichers etc).

Don't bother changing it in this PR, as it's unrelated to everything else. And you've tested it as-is.

I presume you've tested this modified prometheus assertion?